Docker-FAQ.md

```diff
@@ -18,11 +18,11 @@ For some use-cases it is convenient to provide a modified `solr.in.sh` file to S
 For example to point Solr to a ZooKeeper host:
 
 ```
-docker create --name my-solr -P solr
-docker cp my-solr:/opt/solr/bin/solr.in.sh .
+docker create --name my_solr -P solr
+docker cp my_solr:/opt/solr/bin/solr.in.sh .
 sed -i -e 's/#ZK_HOST=.*/ZK_HOST=cylon.lan:2181/' solr.in.sh
```

Comment:
For the purposes of this particular patch, this particular change changes the container name to use underscore instead of a dash. I think I actually ran into some reason that the dashed container name caused me a problem, but I don't recall what that was now. Perhaps I just changed this for consistency with Docker's own namesgenerator scheme, and the other examples in the README. In any case, that is fine.

That FAQ question is specifically about persisting config, and the answer illustrates how to write config to the container. If we use `-e` then there is no persisting, and the answer thus becomes non-sensical. Perhaps we can come up with a better use-case.